streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/ImplicitConversions.scala

```diff
@@ -35,9 +35,9 @@ import org.apache.kafka.streams.scala.kstream._
 import scala.language.implicitConversions
 
 /**
- * Implicit conversions between the Scala wrapper objects and the underlying Java
- * objects.
- */
+  * Implicit conversions between the Scala wrapper objects and the underlying Java
+  * objects.
+  */
```

Comment:
I like this still because it's the default in Scala but it seems that in this code base we adopted the Java style.
So for consistency I guess we should leave it to Java style or bigbang the whole codebase to Scala style